backend/battle/forms.py

```diff
@@ -59,21 +59,26 @@ def clean(self):
         pokemon_1 = self.cleaned_data.get('pokemon_1')
         pokemon_2 = self.cleaned_data.get('pokemon_2')
         pokemon_3 = self.cleaned_data.get('pokemon_3')
-        obj_battle = cleaned_data['battle']
         obj_trainer = cleaned_data['trainer']
-        position_pkn_1 = cleaned_data['position_pkn_1']
-        position_pkn_2 = cleaned_data['position_pkn_2']
-        position_pkn_3 = cleaned_data['position_pkn_3']
+
+        if 'battle' not in cleaned_data:
+            raise forms.ValidationError('ERROR: Select a valid battle')
+
+        obj_battle = cleaned_data['battle']
 
         if not pokemon_1 or not pokemon_2 or not pokemon_3:
             raise forms.ValidationError('ERROR: Select all pokemons')
 
-        if not position_pkn_1 or not position_pkn_2 or not position_pkn_3:
+        if ('position_pkn_1' or 'position_pkn_2' or 'position_pkn_3') not in cleaned_data:
```

Comment:
Same here

Reply:
When I submit the form without pokemons value I already receive the message "The filed is required". But I wanted to create this validation to verify on test

Reply:
if this is already on the form, you shouldn't need to add this extra validation :thinking: